docs/administration/configuring-openidc.md

```diff
@@ -0,0 +1,103 @@
+#Configuring OpenId Connect (OIDC)
+Instruction on how to configure Openid Connect authentication:
+1. When configuring a new Yoda instance
+2. On an already existing Yoda instance
+
+## 1. When configuring a new Yoda instance
+When creating a new Yoda instance, setup the variables in the group_vars as explained in [Configuring Yoda](configuring-yoda.md).
+Alternatively, you can choose to pass the variables with the *--extra-vars* option everytime when running the Ansible playbook.
+
+For OIDC to function properly it requires the following variables to be set:
+- oidc_active
+- oidc_client_id
+- oidc_client_secret
+- oidc_auth_base_uri
+- oidc_token_uri
+- oidc_userinfo_uri
+
+Additionally, depending on the authorization server, you may need to configure the following variables:
+- oidc_scopes (default: *openid*)
+- oidc_acr_values
+- oidc_email_field (default: *email*)
+
+Finally, for customization purposes, you can also configure:
+- oidc_signin_text
+
+## 2. On an already existing Yoda instance
+When configuring OIDC on an already existing Yoda instance the same variables mentioned in the previous section are needed. 
+Whereas the passing them before deploying puts them automatically in the correct places, in this case they need to be set in different locations manually.
```

Comment:
Running the playbook with the OIDC variables configured would set everything automatically in the correct place?

Reply:
Correct. I was unsure whether running the playbook again might undo some things so that is why this was documented.

Reply:
Yeah, we should be able to run the playbook n times and nothing should change.